config/crd/bases/argoproj.io_applications.yaml

```diff
@@ -0,0 +1,90 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Haven't found the right combo of markers to keep this file from being generated while ensuring deepcopy still gets generated. https://book.kubebuilder.io/reference/markers/crd